src/components/covidForm.vue

```diff
@@ -295,6 +316,11 @@ export default {
   name: "covidForm",
   props: ["entity", "entityCheckIn"],
   data() {
+    const validRoles = ["admin", "user"];
+    const showDetails =
+      validRoles.indexOf(this.$jwt.decode(this.$root.auth_token).type) > -1
```

Comment:
Dont think the ternary is necessary here since 
validRoles.indexOf(this.$jwt.decode(this.$root.auth_token).type) > -1 is truthy

Reply:
Good call. Fixed here and from where I copied it from.